src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPayment = 40;
+  const basicDiscount = 50;
+  const minDiscount = 20;
+  const totalCost = days * dayPayment;
+
+  switch (true) {
```

Comment:
we agreed to avoid using switch-case in such cases

Reply:
Replaced